### PR TITLE
Vinagesh/mergemaster

### PR DESCRIPTION
**Pull request by vinagesh**
Merging master to preview

### CODE REVIEW

File: device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java [hunk 1]
```diff
@@ -3,10 +3,12 @@
 
 package samples.com.microsoft.azure.sdk.iot.device;
 
+import com.google.gson.Gson;
 import com.microsoft.azure.sdk.iot.deps.twin.TwinCollection;
 import com.microsoft.azure.sdk.iot.device.*;
 import com.microsoft.azure.sdk.iot.device.DeviceTwin.*;
 import com.microsoft.azure.sdk.iot.pnphelpers.PnpHelper;
+import lombok.NonNull;
```
Comment (bikamani) [inline device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java:11]: is this used?

Comment (vinagesh) [reply device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java:11]: It came in with merge. Let me check.

**Pull request merged**